gremlin-dotnet/glv/P.template

```diff
@@ -94,8 +94,8 @@ namespace Gremlin.Net.Process.Traversal
 <% } %><% pmethods.findAll{ it in ["within", "without"] }.each { method -> %>
         public static P <%= toCSharpMethodName.call(method) %>(params object[] args)
         {
-            if (args.Length == 1 && args[0] is ICollection<object>)
-                return new P("<%= method %>", ToGenericArray((ICollection<object>) args[0]));
+            if (args.Length == 1 && args[0] is ICollection<object> collection)
+                return new P("without", ToGenericArray(collection));
```

Comment:
I don't think the method should be hard-coded as `without` here 😄

I just debugged one of the failing tests and was really confused when I saw `P.without()` in the Bytecode although the scenario only uses `P.within()`, but good that we now have such a good test coverage with the Gherkin features that we immediately find slips like this one 😅

Reply:
Nice catch! I've looked at it and missed it completely :)

Reply:
hahaha....oops - copy/paste. thanks-that was dumb on my part